Dockerfile

```diff
@@ -50,23 +34,12 @@ RUN apk add --no-cache \
     && rm -rf /var/cache/apk/*
 
 COPY build/                   /app/build
-#
-# @deprecated
-# to be deleted in PRMT-4588
-# COPY database/                /app/database
-# COPY build/config/database.js /app/src/config/
-# COPY .sequelizerc             /app/
-
-COPY scripts/run-server-with-db.sh /usr/bin/run-ehr-out-service
+
+COPY scripts/run-server.sh /usr/bin/run-ehr-out-service
 COPY scripts/load-api-keys.sh      /app/scripts/load-api-keys.sh
 
 ENV NHS_ENVIRONMENT="" \
```

Comment:
Is this required - no value is set? Can't we just assign the environment variables when spinning an instance up?